content/actions/managing-issues-and-pull-requests/commenting-on-an-issue-when-a-label-is-added.md

```diff
@@ -30,38 +30,38 @@ In the tutorial, you will first make a workflow file that uses the [`peter-evans
 1. Copy the following YAML contents into your workflow file.
 
     ```yaml copy
-    {% data reusables.actions.actions-not-certified-by-github-comment %}
-
-    {% data reusables.actions.actions-use-sha-pinning-comment %}
-
     name: Add comment
     on:
       issues:
         types:
           - labeled
     jobs:
       add-comment:
-        if: github.event.label.name == 'help-wanted'
+        if: github.event.label.name == 'help wanted'
         runs-on: ubuntu-latest
         permissions:
           issues: write
         steps:
           - name: Add comment
-            uses: peter-evans/create-or-update-comment@5f728c3dae25f329afbe34ee4d08eef25569d79f
-            with:
-              issue-number: {% raw %}${{ github.event.issue.number }}{% endraw %}
-              body: |
-                This issue is available for anyone to work on. **Make sure to reference this issue in your pull request.** :sparkles: Thank you for your contribution! :sparkles:
+            # https://cli.github.com/manual/gh_issue_comment
```

Comment:
It's up to the Docs reviewers but I think this is a bit unnecessary given that you're on the documentation already. 
If you can use `gh issue ...` you, in a sense, don't need the manual.